tests/integration_tests/cdc_server_tips/run.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -eu
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=cdc.test
+SINK_TYPE=$1
+stdout_file=$WORK_DIR/stdout.log
+cdc_launched=
+
+function try_to_run_cdc() {
+	# $*: invalid args to avoid lauching cdc server
+	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
+	start_tidb_cluster --workdir $WORK_DIR
```

Comment:
What about moving `start_tidb_cluster` out of `try_to_run_cdc`, starting tidb cluster once and reusing it.

Reply:
Thx!